lib/metasearch_index/deleter.rb

```diff
@@ -2,18 +2,16 @@ module MetasearchIndex
   module Deleter
     class V2
       class MissingArgument < StandardError; end
```

Comment:
Could we just use the standard ArgumentError here?

Reply:
yes, that makes more sense..